vpc.yml

```diff
@@ -20,7 +20,7 @@
         region: "{{ aws_region }}"
         vpc_id: "{{ vpc_id }}"
         tags:
-          Name: "{{ vpc_name }}"
+          Name: "{{ vpc_name }}-igw"
```

Comment:
Not sure if we actually need it since we are going to have just one default internet gateway.... so it would be just nicer to see VPC name in the UI. but up to you